star-wars.Rmd

```diff
@@ -5,16 +5,31 @@ output: github_document
 
 ```{r setup, include = FALSE}
 knitr::opts_chunk$set(echo = FALSE)
+library(dplyr)
+library(tidyr)
+library(knitr)
 ```
 
 ## Most accomplished
 
 Below we look at which 5 characters from Star Wars are the most accomplished pilot (flew the fewest number of starships):
 
 ```{r table-of-most-accomplished-pilots}
+accomp_pilots <- starwars %>% 
+  unnest(films) %>% 
+  group_by(name) %>% 
+  tally() %>% 
+  arrange(desc(n)) %>% 
+  head(10)
+
+kable(accomp_pilots,
+  col.names = c("Character", "Number of Starships piloted"))
 ```
 
-By far, the most accomplished pilot of all is ????. He is shown below piloting a starship.
+By far, the most accomplished pilot of all is `r accomp_pilots$name[1]`. He is shown below piloting a starship.
 
 ```{r top-pilot, fig.cap="Most accomplished Star Wars pilot.", out.width = "100%"}
+include_graphics("https://starwarz.com/tbone/wp-content/uploads/2011/08/331.jpg")
```

Comment:
Because of the error above, we need a different, picture, the accomplished pilot is Obi-Wan Kenobi:

```suggestion
include_graphics("https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/ewan-mcgregor-obi-wan-1570898048.jpg")
```